connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectWorkerIntegrationTest.java

```diff
@@ -771,6 +779,114 @@ private Map<String, String> defaultSinkConnectorProps(String topics) {
         return props;
     }
 
+    @Test
+    public void testRequestTimeouts() throws Exception {
+        final String configTopic = "test-request-timeout-configs";
+        workerProps.put(CONFIG_TOPIC_CONFIG, configTopic);
+        // Workaround for KAFKA-15676, which can cause the scheduled rebalance delay to
+        // be spuriously triggered after the group coordinator for a Connect cluster is bounced
+        // Set to 1 instead of 0 as another workaround for KAFKA-15693, which can cause
+        // connectors and tasks to be unassigned indefinitely if the scheduled rebalance delay
+        // is set to 0
```

Comment:
I've just noticed that this part is no longer necessary since we've merged a [fix](https://github.com/apache/kafka/pull/14647) for [KAFKA-15693](https://issues.apache.org/jira/browse/KAFKA-15693).

I'll remove this comment and change the value to zero. If the test continues to pass on Jenkins, should be safe to merge.